### PR TITLE
RUN-51 Add Links

### PR DESCRIPTION
**Pull request by Jnwong116**
## Changes

- Created the links constant
- Updated FB button styling
- Created links section/navbar at the top w placeholder links
- Added key attributes to various components

## Checklist

- [ ] Pull request has descriptive context useful to reviewer. Screenshots are attached as necessary.
- [ ] Pull request title matches Linear ticket.
- [ ] Ticket has been claimed on Linear
- [ ] Pull Request has been assigned to yourself
- [ ] Pull request has been approved by CodeOwner
- [ ] Pull request has been approved by one other student

### CODE REVIEW

File: client/src/component/facebookLoginButton/facebookLoginButton.js [hunk 1]
```diff
@@ -18,10 +20,10 @@ class FacebookLoginButton extends React.Component {
         isMobile={false}
         fields="name,picture"
         callback={responseFacebook}
-        render={(renderProps) => (
-          <button className="fb-login-button" onClick={renderProps.onClick}>
+        render={() => (
```
Comment (Jnwong116) [inline client/src/component/facebookLoginButton/facebookLoginButton.js]: Why did u change this? Also you need to include the `onClick`

File: client/src/component/candidateFlyout/candidateFlyout.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ const CandidateFlyout = ({ person, flyoutOpen, setFlyoutOpen }) => {
         {hasNominators &&
           nominations.map((nom) => (
             <WhiteContainer
+              key={nom.nominator}
```
Comment (Jnwong116) [inline client/src/component/candidateFlyout/candidateFlyout.js:63]: What is this for as well?

**Comment by Jnwong116** [comment]
Also you need to approve it @marianmwang

File: client/src/component/facebookLoginButton/facebookLoginButton.js (hunk 1 above)
Comment (marianmwang) [reply client/src/component/facebookLoginButton/facebookLoginButton.js]: so we use mui components :,) there's also some styling already applied by the mui so it's more consistent

File: client/src/component/candidateFlyout/candidateFlyout.js (hunk 2 above)
Comment (marianmwang) [reply client/src/component/candidateFlyout/candidateFlyout.js:63]: react thing, every child in a list needs a unique key so it can do its magic

**Pull request merged**